### PR TITLE
ION create operation FFI

### LINKED ISSUE #117: Generate create operation deterministically from mnemonic
Construct a new ION DID from a mnemonic allowing transfer between mobile and desktop to facilitate [verifiable presentations](https://github.com/alan-turing-institute/trustworthy-id/issues/580).

### PR DESCRIPTION
**Pull request by sgreenbury**
Closes #117.

### CODE REVIEW

File: trustchain-ffi/Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,7 @@ trustchain-ion = { path = "../trustchain-ion" }
 trustchain-api = { path = "../trustchain-api" }
 
 anyhow = "1.0"
+bip39 = "2.0.0"
```
Comment (sgreenbury) [inline trustchain-ffi/Cargo.toml:18]: No longer required here.

File: trustchain-ion/src/create.rs [hunk 2]
```diff
@@ -174,6 +283,7 @@ mod test {
         for path in paths {
             if let Ok(path_buf) = path {
                 let operation_string = std::fs::read_to_string(path_buf)?;
+                println!("{}", operation_string);
```
Comment (sgreenbury) [inline trustchain-ion/src/create.rs:286]:
```suggestion
```

File: trustchain-ffi/Cargo.toml (hunk 1 above)
Comment (sgreenbury) [reply trustchain-ffi/Cargo.toml]: Removed

**Review by thobson88 — APPROVED** [review]
LGTM!

**Pull request merged**